source/includes/examples/_turtlebot.html.md

```diff
@@ -0,0 +1,174 @@
+
+###### Concept
+
+```inkling
+concept go_to_point is estimator
+    predicts (Command)
+    follows input(State)
+    feeds output
+end
+```
+
+This concept is named `go_to_point` which predicts a `Command` given a `State`
+of the simulation. This concept of `go_to_point` is based on the x, y position
+set as the goal.
+
+###### Simulator
+
+```inkling
+simulator turtlebot_office_simulator(TurtlebotConfig)
+    action (Command)
+    state (State)
+end
+```
+
+
+The simulator clause declares that a simulator named `turtlebot_office_simulator`
+will be connecting to the server for training. This code snippet binds the
+previous schemas to this simulator. To define the training relationship between
+the simulator and the concept we must begin by defining the simulator.
+`turtlebot_office_simulator` expects an action defined in the `Command` schema
+as input and replies with a state defined in the `State` schema as output.
+
+###### Curriculum
+
+```inkling
+curriculum go_to_point_curriculum
+    train go_to_point
+    with simulator turtlebot_office_simulator
+    objective get_reward
+        lesson get_to_point
+            configure
+                constrain unused with Int8{-1}
+            until
+                maximize get_reward
+end
+```
+
+The curriculum `go_to_point_curriculum` trains `go_to_point` using
+`turtlebot_office_simulator`. The BRAIN that runs this Inkling code will try to
+maximize the value returned from `get_reward` until you stop training.
+`get_reward` is a method in the Python simulator.
+
+This curriculum contains one lesson, called `get_to_point`. If used, it would
+configure the simulation by setting a number of constraints for the state of
+the simulator. In this simple example however, there are no constraints.
+
+## Simulator Files
+
+The three full Python files for this example are with the rest of the
+[gazebo-turtlebot-sample code][1] on GitHub.
+
+### bonsai_ros.py
+
+> Source code too long to display, please click Github link.
+
+This Python file mainly contains a class that acts as a basic interface between
+ROS and Bonsai's protocols. It can bring up and down the ROS stack and parse a
+variety of basic messages.
+
+The purpose of this file is to manage the interactions between the BRAIN server
+interface and many ROS topics that are available to subscribe or publish to.
+
+See the [**bonsai_ros.py file**][6] on GitHub.
+
+### bonsai_gazebo.py
+
+> Source code too long to display, please click Github link.
+
+This Python file extends the BonsaiROS interface with methods for controlling
+Gazebo simulations in accordance with the Bonsai simulator protocol.
+
+The purpose of this file is to extend `bonsai_ros.py` to also handle interacting
+with the Gazebo specific services and topics made available when using a Gazebo
+simulation with ROS.
+
+See the [**bonsai_gazebo.py file**][5] on GitHub.
+
+### turtlebot_office_experiment.py
+
+> Source code too long to display, please click Github link.
+
```

Comment:
This is very subjective, but perhaps starting off with something that emphasizes this is the simulator functionality that drives AI training and prediction.